README.md

```diff
@@ -1,51 +1,52 @@
 # Nerves.WpaSupplicant
 [![Build Status](https://travis-ci.org/nerves-project/nerves_wpa_supplicant.svg?branch=master)](https://travis-ci.org/nerves-project/nerves_wpa_supplicant)
 
-This package enables Elixir applications to interact with the local WPA
-supplicant. The WPA supplicant handles various Wi-Fi operations like scanning
-for wireless networks, connecting, authenticating, and collecting wireless
-adapter statistics.
+This package enables Elixir applications to query or effect changes on Wi-Fi
+network connections of a host system, by interacting with a [wireless
+supplicant].
 
-*This library is under development and we plan on making the API more user friendly in future versions*
+The portable [wpa_supplicant] daemon handles Wi-Fi operations like scanning for
+wireless networks, connecting, authenticating, and collecting wireless adapter
+statistics. `Nerves.WpaSupplicant` uses the control interface provided by this
+supplicant implementation to bring these capabilities to your Elixir code.
```

Comment:
Actually if you don't like these two paragraphs for some reason, the existing Hex package description is a good alternative superior to the original text here, I think.